Arrowgene.Ddon.Shared/Files/Assets/quests/q20055004 Redcaps.json

```diff
@@ -3,6 +3,7 @@
   "type": "World",
   "comment": "The Abductors’ True Nature",
   "quest_id": 20055004,
+  "variant_id": 121231,
```

Comment:
Are variant ids just randomly picked numbers? How were they generated. The game also has some concept of distribution id. We currently just assign it the quest id, but maybe this is actually the use case for it?